content/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -18,6 +18,8 @@ You can manage run tasks through the UI as detailed below or through the [Run Ta
 
 ## Creating a Run Task
 
+Refer to [Run Tasks Integration](/cloud-docs/integrations/run-tasks#prerequisites) for a list of available run tasks. These include the HCP Packer integration and run tasks developed by HashiCorp Technology Partners. 
```

Comment:
Should this go to a subheading (probably `#hcp-packer-run-task`) rather than the top of the page?

Reply:
yeah you're right - thank you! I'm going to add that in and then merge! :-)